Scarab/Services/Installer.cs

```diff
@@ -12,7 +12,6 @@
 using Avalonia.Threading;
 using MsBox.Avalonia.Enums;
 using Microsoft.Toolkit.HighPerformance;
-using MsBox.Avalonia.Enums;
```

Comment:
Could you undo this. I have local changes that would conflict with this

Reply:
done